test/merger/test_general_merger.py

```diff
@@ -26,7 +31,7 @@ def test_compose_bloch_sphere_rotations_different_axis() -> None:
     assert composed == BlochSphereRotation(qubit=123, axis=(1, 1, 0), angle=math.pi)
 
 
-def test_single_gate() -> None:
+def test_single_gate(merger: Merger) -> None:
```

Comment:
This can be more specific
```suggestion
def test_single_gate(merger: SingleQubitGatesMerger) -> None:
```